vms/platformvm/network/gossip_test.go

```diff
@@ -84,14 +116,37 @@ func TestMempoolDuplicate(t *testing.T) {
 func TestGossipAddBloomFilter(t *testing.T) {
 	require := require.New(t)
 
+	avaxAssetID := ids.GenerateTestID()
```

Comment:
We expose a `snowtest.AVAXAssetID` which is more compatible with the rest of our test helpers. Could we use that rather than generating a new ID?